src/ipmi.c

```diff
@@ -275,10 +306,17 @@ static int sensor_list_add(ipmi_sensor_t *sensor) {
     type = "fanspeed";
     break;
 
-  default: {
-    const char *sensor_type_str;
+  case IPMI_SENSOR_TYPE_MEMORY:
```

Comment:
Can you please show 'ipmitool' output for such a sensor?
For example, by running `sensor get SENSOR_NAME` ?

Thanks.

Reply:
Hi @rpv-tomsk, see me comment above for this example https://github.com/collectd/collectd/pull/2071#issuecomment-296641468

Reply:
In nutshell, this sensor represents "Memory Thermal Throttling" and is related to memory thermal management system. Based on the DIMM thermal conditions it may restrict read and write traffic/bandwidth to main memory as a means of controlling power consumption. This metric is measured as a percentage and 0% means no memory throttling occurs. When thermal conditions are going high, the memory management system enables throttling and restricts the read or write traffic (e.g. 50%).